template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/Manage/TwoFactorAuthentication.cshtml

```diff
@@ -5,44 +5,32 @@
 }
 
 <h4>@ViewData["Title"]</h4>
-@if (Model.HasAny2faProviders)
+@if (Model.HasAny2faProviders && Model.Is2faEnabled)
```

Comment:
I also think renaming things to make them sound authenticator specific is misleading. There is no notion of an authenticator app being "enabled" in the runtime. Currently the names directly reflect the runtime state, which is clear and correct.

We could however just remove the HasAny2faProviders property. I wasn't sure if it was needed anymore. @HaoK is checking `Is2faEnabled` sufficient in this case?